### ISSUE TITLE
Search field in uiowa.edu 

### ISSUE THREAD

**Issue by inspector508** [uiowa/uiowa#1315]
Guessing this goes to Ben S.??
When a user expands the search region, the actual search field is several tab stops backward. Can we send focus to the search form field on expand?

**Comment by bspeare** [comment]
@inspector508 I think that fix is coming in an update later today.  I'll ping you to verify when it is up on the stage site.

**Comment by bspeare** [comment]
@inspector508 This fix should be up on stage now.
